builder/assets/butterfly.service

```diff
@@ -2,5 +2,5 @@
 Description=Butterfly Terminal Server
 
 [Service]
-ExecStart=/usr/local/bin/butterfly.server.py --host="0.0.0.0" --unsecure
+ExecStart=/bin/bash -c ". /root/butterfly_env/bin/activate; butterfly.server.py --host="0.0.0.0" --unsecure"
```

Comment:
Wouldn't it be better to just use Python 2 for Butterly?

Reply:
Might be, though I'd really like to keep as little of Python 2 as possible. In fact, we could probably just use a lower version of Tornado for Butterfly, it shouldn't cause any problems.

Reply:
I've restored system-wide butterfly installation, it *should* work with Tornado 4.2.1.